src/PrefetchTemplate.js

```diff
@@ -44,35 +47,43 @@ export class PrefetchTemplate {
     paramElementMap.set('context.draftOrders.context.appointments.Appointment.id', ['id']);
     paramElementMap.set('context.draftOrders.context.encounterId', ['id']);
     paramElementMap.set('context.patientId', ['subject', 'reference']);
+    paramElementMap.set('context.pharmacyId', ['id']); 
+
     return paramElementMap;
   }
 
-  static generateQueries(requestBundle, patientReference, userReference, ...prefetchKeys) {
-    var resolvedQueries = new Map();
-    for (var i = 0; i < prefetchKeys.length; i++) {
-      var prefetchKey = prefetchKeys[i];
-      var query = prefetchMap.get(prefetchKey).getQuery();
+  static generateQueries(requestBundle, patientReference, userReference, pharmacyId, ...prefetchKeys) {
+  var resolvedQueries = new Map();
+  for (var i = 0; i < prefetchKeys.length; i++) {
+    var prefetchKey = prefetchKeys[i];
+    var query = prefetchMap.get(prefetchKey).getQuery();
       // Regex source: https://regexland.com/all-between-specified-characters/
-      var parametersToFill = query.match(/(?<={{).*?(?=}})/gs);
-      var resolvedQuery = query.slice();
+    var parametersToFill = query.match(/(?<={{).*?(?=}})/gs);
+    var resolvedQuery = query.slice();
       for (var j = 0; j < parametersToFill.length; j++) {
         var unresolvedParameter = parametersToFill[j];
         var resolvedParameter;
         if (requestBundle) {
-          resolvedParameter = PrefetchTemplate.resolveParameter(unresolvedParameter, requestBundle);
+          if (unresolvedParameter === 'context.pharmacyId') {
```

Comment:
Do we send the pharmacy information with every request? It would be good to have examples with and without since the pharmacy is optional and not required.